backend/DB/migrations/20211017182518-create-intake-form.cjs

```diff
@@ -0,0 +1,34 @@
+"use strict";
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable("IntakeForms", {
```

Comment:
I wonder if we'll need a way to distinguish between internal intake forms and external intake forms, or other kinds of intake forms when there are multiple for the same noun. Not sure if we need to do anything for that now, or not.